src/main/java/wellnus/storage/ReflectionTokenizer.java

```diff
@@ -78,62 +98,57 @@ private Set<Integer> getSet(String indexToSplit) throws TokenizerException {
         } catch (NumberFormatException numberFormatException) {
             throw new TokenizerException(DETOKENIZE_ERROR_MESSAGE);
         }
+        if (categoryKey.equals(PREV_KEY) && outputIndexes.size() > MAX_NUM_PREV_INDEX) {
```

Comment:
this is only applicable to prev list, prev list size need to be exactly 5, less than or greater than 5 are not allowed.